R/data_kara.R

```diff
@@ -0,0 +1,49 @@
+#' NBA Elo Ratings Through History
+#' 
+#' The data behind the Elo and RAPTOR ratings for NBA teams used in various stories like this one: 
+#' \url{https://projects.fivethirtyeight.com/complete-history-of-the-nba/#raptors}
+#' 
+#'   @format A data frame with 69636 rows representing every game in the NBA, and 24 variables:
+#' \describe{
+#'   \item{date}{Date}
+#'   \item{season}{the season in which the game was played}
+#'   \item{neutral}{1 if the game was played on neutral territory, 0 if not}
+#'   \item{playoff}{1 if the game was played in a playoff, o if not}
+#'   \item{team1}{name of first team}
+#'   \item{team2}{name of second team}
+#'   \item{elo1_pre}{Team 1 Elo rating before game}
+#'   \item{elo2_pre}{Team 2 Elo rating before game}
+#'   \item{elo_prob1}{Team 1's probability of winning based on Elo rating}
+#'   \item{elo_prob2}{Team 2's probability of winning based on Elo rating}
+#'   \item{elo1_post}{Team 1 Elo rating after the game}
+#'   \item{elo2_post}{Team 2 Elo rating after the game}
+#'   \item{score1}{the score of team 1}
+#'   \item{score2}{the score of team 2}
+#'   }
+#'   
+"nba_elo"
+
+#' NBA Elo Ratings for this season
```

Comment:
Also, since these two data frames are very related, it makes sense to combine them into the same help file. See this example: https://github.com/rudeboybert/fivethirtyeight/blob/master/R/data_marium.R